### PR TITLE
S3#5667 YouTube All Over The Place...

### PR DESCRIPTION
**Pull request by Sk0vgaard**

### CODE REVIEW

File: app/src/main/java/dk/adamino/rehabilitation/GUI/ExerciseListActivity.java [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+        return intent;
+    }
+
+    /**
+     * The client found in Firestore.
+     * @param clientFound
+     */
+    @Override
+    public void onClientResponse(Client clientFound) {
+        Log.d("UpdateUI", "ClientResponse");
```
Comment (onero) [inline app/src/main/java/dk/adamino/rehabilitation/GUI/ExerciseListActivity.java]: @MJ1227UB  We no LIKEY CONSOLY!

File: app/src/main/java/dk/adamino/rehabilitation/GUI/ProfileActivity.java [hunk 2]
```diff
@@ -60,6 +61,8 @@ private void setClientInformation(Client loggedInClient) {
         // Rehabilitation Plan
         mDiagnosis.setText(loggedInClient.rehabilitationPlan.diagnosis);
         mGoal.setText(loggedInClient.rehabilitationPlan.goal);
+
+        Log.d("LoggedInClient", loggedInClient.rehabilitationPlan.exerciseIds.get(0));
```
Comment (onero) [inline app/src/main/java/dk/adamino/rehabilitation/GUI/ProfileActivity.java]: MOAR LOGS!?

Comment (Sk0vgaard) [inline app/src/main/java/dk/adamino/rehabilitation/GUI/ProfileActivity.java]: @onero Log? :P

File: app/src/main/java/dk/adamino/rehabilitation/GUI/ExerciseListActivity.java (hunk 1 above)
Comment (MJ1227UB) [reply app/src/main/java/dk/adamino/rehabilitation/GUI/ExerciseListActivity.java]: gone! ^^;

**Pull request merged**